app/controllers/content_items_controller.rb

```diff
@@ -27,9 +30,14 @@ def set_guide_draft_access_token
 
   def load_content_item
     content_item = content_store.content_item(content_item_path)
+    raise SpecialRouteReturned if special_route?(content_item)
     @content_item = present(content_item)
   end
 
+  def special_route?(content_item)
+    content_item && content_item['document_type'] == "special_route"
```

Comment:
Yeah... maybe. I think that would be an 'actual exception' though rather than something we know is happening and need to handle.